test/fixtures/transaction.json

```diff
@@ -802,7 +672,7 @@
       "value": 600000000
     },
     {
-      "description": "P2SH(P2WPKH) with SIGHASH_ALL",
+      "description": "P2WPKH with SIGHASH_ALL",
       "txHex": "0100000001db6b1b20aa0fd7b23880be2ecbd4a98130974cf4748fb66092ac4d3ceb1a54770100000000feffffff02b8b4eb0b000000001976a914a457b684d7f0d539a46a45bbc043f35b59d0d96388ac0008af2f000000001976a914fd270b1ee6abcaea97fea7ad0402e8bd8ad6d77c88ac92040000",
```

Comment:
@afk11 indeed, but `hashForWitnessV0` doesn't care.
It just takes the types of arguments it takes, it doesn't care about the context.
The fixtures should reflect that where possible.